magicbyirineuTests/Modules/CardList/CardListInteractorSpec.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  CardListInteractorSpec.swift
+//  magicbyirineuTests
+//
+//  Created by kaique.magno.santos on 01/03/19.
+//  Copyright © 2019 DanielLima. All rights reserved.
+//
+
+import Quick
+import Nimble
+import Nimble_Snapshots
+
+@testable import magicbyirineu
+
+class CardListInteractorSpec:QuickSpec{
+    
+    override func spec() {
+        
+        let cardRepositoryMock = CardRepositoryMock()
+        let cardSetRepositoryMock = CardSetRepositoryMock()
+        let typeRepositoryMock = TypeRepositoryMock()
+        
+        var sut:CardListInteractor!
+        
+        beforeEach {
+            sut = CardListInteractor(cardRepository: cardRepositoryMock, cardSetRepository: cardSetRepositoryMock, typeRepository: typeRepositoryMock)
+        }
+        
+        context("When it is initialized"){
```

Comment:
Todos os testes já escritos estamos usando letra minúscula no início da sentença do contexto e dos it's